packages/thumbprint-react/package.json

```diff
@@ -63,5 +63,9 @@
         "rollup-plugin-typescript": "^1.0.1",
         "tslib": "^1.10.0",
         "typescript": "^3.9.6"
+    },
+    "resolutions": {
+        "react": "17.0.2",
+        "react-dom": "17.0.2"
```

Comment:
These shouldn't be in this file because `resolutions` is something that the consumer of a package sets.

If we want to upgrade the version of React used in this codebase, we'd do it by editing the root `package.json` and `www/package.json`. But that can be done in a separate step since it's a different task than indicating that the `@thumbtack/thumbprint-react` package works with React 17.